.github/workflows/snap-build.yaml

```diff
@@ -0,0 +1,21 @@
+name: Snap Build
+on:
+  pull_request:
+    branches: ["master", "release/**"]
+jobs:
+  build:
+    name: Build the snapd snap
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build the snapd snap
+        uses: snapcore/action-build@v1
+      - name: Rename built snap file
+        run: |
+          sudo mv snapd*.snap snapd.snap
+      - name: Uploading snap artifact
+        uses: actions/upload-artifact@v1
```

Comment:
Any special reason to use v1 instead of v2?

Reply:
that was just what the example I was using had :smiley:

Reply:
v2 works the same but also allows globbing, I was using it wrong